references/kvm-admin-api/README.md

```diff
@@ -0,0 +1,48 @@
+# KVM Admin Proxy
+
+Depending on the Apigee deployment model you might have Managment APIs for
+acessing and modifying the content of Apigee Key Value Maps (KVMs). In cases
+where the access to the KVMs is not provided via the Apigee APIs you will need
+to leverage policies to perform CRUD operations on the KVM contents.
+
+This project provides a reference implementation for how to read, write and
+delete entries within environment scoped KVMs inside Apigee regardless of the
+deploment model and the existance of KVM managment APIs.
+
+## (Prerequisite) Create a KVM
+
+**Note:** Apigee SaaS currently does not allow for dynamic KVM names via the
+MapName element. It will automatically default to a KVM called `kvmap` and
+ignore the provided map name in the path.
+
+```bash
```

Comment:
this can be sh to be consistent with the ones below

Reply:
ack